geotrek/v_treks_schema.sql

```diff
@@ -119,24 +135,26 @@ SELECT
     t.access AS acces_routier,
     t.public_transport AS transports,
     advised_parking AS parking_info,
-    ST_AsText(st_snaptogrid(st_transform(parking_location, 4326), 0.000027::double precision)) AS parking_geometrie,
+    -- Reprojection des géométries en 4326
+    ST_AsText(st_transform(parking_location, 4326)) AS parking_geometrie,
     date(top.date_insert)::text AS date_creation,
     date(top.date_update)::text AS date_modification,
     medias.liste AS medias,
-    parent.parent_id::varchar AS itineraire_parent,
+    parent.liste::text AS itineraire_parent,
     sol.liste::text AS type_sol,
     NULL::boolean AS pdipr_inscription,
     NULL::text AS pdipr_date_inscription,
-    -- réduction de la précision des coordonnées à 5 décimales, simplification de la géométrie pour réduire le nombre de points. Poids de la géométrie divisé par 7.5
-    st_simplifypreservetopology(st_snaptogrid(st_transform(top.geom, 4326), 0.000027::double precision), 0.000027::double precision) AS geom
+    -- Reprojection des géométries en 4326
+    st_transform(top.geom, 4326) AS geom
 FROM selected_t t
 JOIN core_topology top ON t.topo_object_id = top.id
 JOIN sources ON t.topo_object_id = sources.trek_id
```

Comment:
```suggestion
LEFT JOIN sources ON t.topo_object_id = sources.trek_id
```
comme tu l'as suggéré